akka-stream/src/main/scala/akka/stream/impl/fusing/GraphStages.scala

```diff
@@ -37,8 +37,8 @@ object GraphStages {
    * INERNAL API
    */
   private[stream] abstract class SimpleLinearGraphStage[T] extends GraphStage[FlowShape[T, T]] {
-    val in = Inlet[T]("in")
-    val out = Outlet[T]("out")
+    val in = Inlet[T](toString + ".in")
+    val out = Outlet[T](toString + ".out")
```

Comment:
Does: `Logging.simpleName(this) + ".in"` and `Logging.simpleName(this) + ".out"` work?